packages/react-native-reanimated/compatibility.json

```
@@ -1,44 +1,58 @@
 {
   "nightly": {
-    "react-native": ["0.78", "0.79", "0.80", "0.81"]
+    "react-native": ["0.78", "0.79", "0.80", "0.81"],
+    "react-native-worklets": ["0.4.x"]
   },
   "4.0.x": {
-    "react-native": ["0.78", "0.79", "0.80", "0.81"]
+    "react-native": ["0.78", "0.79", "0.80", "0.81"],
+    "react-native-worklets": ["0.4.x"]
   },
   "3.19.x": {
-    "react-native": ["0.78", "0.79", "0.80", "0.81"]
+    "react-native": ["0.78", "0.79", "0.80", "0.81"],
+    "react-native-worklets": ["0.4.x"]
```

Reanimated 3 doesn't work with Worklets.

```suggestion
    "react-native-worklets": []
```